vercel/resource_deployment.go

```diff
@@ -308,7 +308,26 @@ func (r resourceDeployment) Update(ctx context.Context, req tfsdk.UpdateResource
 // Delete does nothing other than clear the existing terraform state for a Deployment.
 // This is done intentionally, as typically, Vercel users do not continually delete old Deployments.
 func (r resourceDeployment) Delete(ctx context.Context, req tfsdk.DeleteResourceRequest, resp *tfsdk.DeleteResourceResponse) {
-	tflog.Trace(ctx, "deleted deployment")
+	//err := r.p.client.DeleteProject(ctx, state.ID.Value, state.TeamID.Value)
+	var state Deployment
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	dResp, err := r.p.client.DeleteDeployment(ctx, state.URL.Value, state.TeamID.Value)
```

Comment:
What's the rationale behind using URL over deploymentID?

Reply:
I thought the Id was not included in the create deployment response, I just double checked and it does, so I'll change it

Reply:
changed